lib/polishgeeks/dev-tools/commands/base.rb

```diff
@@ -57,6 +57,16 @@ def ensure_executable!
             validator_class.new(stored_output).validate!
           end
         end
+
+        # @param [String] path from which we want take files
+        # @return [Array<String>] list of files in path with app prefix path
+        # @note if path is a file return array with file path with app prefix path
+        def files_from_path(path)
```

Comment:
shouldnt this method be private? we dont use it as a part of commands api I guess